Products/products.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express')
+const router = express.Router();
+
+const {Product} = require("./product.model.js")
+
+router.route("/")
+  .get(async(req,res) => {
+
+    const products = await Product.find({})
+    res.status(200).json({success: true, products})
```

Comment:
This could be put inside a try-catch block

Reply:
Done